openlane/config/removals.py

```diff
@@ -36,4 +36,10 @@
     "MAGIC_GDS_ALLOW_ABSTRACT": "Bad practice. If an abstract view is needed, a GDS file can be generated from the abstract LEF file.",
     "PLACE_SITE_HEIGHT": "Now automatically extracted from PLACE_SITE.",
     "PLACE_SITE_WIDTH": "Now automatically extracted from PLACE_SITE.",
+    "LEC_ENABLE": "Complex, unscalable, and largely reundant.",
```

Comment:
Shouldn't `LEC_ENABLE` a deprecated variable for `RUN_EQY`

Reply:
No, it behaves fairly differently